crates/service-client/Cargo.toml

```diff
@@ -29,28 +28,30 @@ http = "1.1.0"
 http-serde = "2.0.0"
 http-body-util = "0.1.2"
 hyper = { version = "1.4.1", features = ["http1", "http2", "client"] }
-# TODO Remove once bumped lambda
-hyper-0-14 = { package = "hyper", version = "0.14.30" }
 hyper-util = { version = "0.1.6", features = ["client-legacy"] }
 hyper-rustls = { version = "0.27.2", features = ["http2"] }
-# TODO Remove once bumped lambda
-hyper-rustls-0-24 = { package = "hyper-rustls", version = "=0.24.1", features = ["http2"] }
 humantime = { workspace = true }
 jsonwebtoken = { version = "9.1.0" }
 once_cell = { workspace = true }
 pem = { version = "3.0.3" }
 tower-service = { version = "0.3" }
 ring = { version = "0.17.8" }
 restate-types = { workspace = true }
-rustls = "0.23.11"
-# TODO Remove once bumped lambda
-rustls_0_21 = { version = "0.21.11", package = "rustls" }
+rustls = {version = "0.23.11"}
```

Comment:
This might break because of a conflicting feature import from another transitive dependency... Let's see if CI passes but will definitely break on my PR here: https://github.com/restatedev/restate/pull/1733

Maybe let's wait my PR first, but I'm sure here you must disable the default features and bring in `ring` instead of the aws feature thingy: https://github.com/restatedev/restate/pull/1733/files#diff-cd07fc573167803c5a91361950d3683cf7e2dc3bfa9c43d901dc8799bda5f506R35

Reply:
let's see if CI passes now though, curious :)

Reply:
it does pass. why do we need to use ring?

Reply:
because reqwest latest version brings in by default `ring`, and i couldn't find a way to switch it to the aws library